iina/AppDelegate.swift

```diff
@@ -135,6 +137,13 @@ class AppDelegate: NSObject, NSApplicationDelegate {
   }
 
   func application(_ sender: NSApplication, openFile filename: String) -> Bool {
+    // When dragging multiple files to IINA icon, cocoa will simply call this method repeatedly.
+    // IINA (mpv) can't handle opening multiple files correctly, so I have to guard it here.
+    // It's a temperory solution, and the min time interval 0.3 might also be too arbitrary.
+    let c = CFAbsoluteTimeGetCurrent()
```

Comment:
I just want a quick way to get a timestamp, rather than creating a powerful but useless `Date` object.